internal/sequencerapi/api.go

```diff
@@ -22,14 +23,16 @@ var (
 )
 
 type sendRawTxCond struct {
-	b      ethapi.Backend
-	seqRPC *rpc.Client
+	b           ethapi.Backend
+	seqRPC      *rpc.Client
+	costLimiter *rate.Limiter
 }
 
-func GetSendRawTxConditionalAPI(b ethapi.Backend, seqRPC *rpc.Client) rpc.API {
+func GetSendRawTxConditionalAPI(b ethapi.Backend, seqRPC *rpc.Client, costRateLimit int) rpc.API {
+	costLimiter := rate.NewLimiter(rate.Limit(costRateLimit), params.TransactionConditionalMaxCost)
```

Comment:
I think you're still a little confused about how burst works. Technically yes, the incoming txs can't exceed the max cos, so burst will enforce this too (altho it's checked elsewhere too), but the way the code is structured you'll get just ONE max cost tx and everything else will get spread out very thin. Normally you set burst a bit higher to allow things to queue up or load up the worker a bit before starting to slow things down. That said, if it's intentional, all good.

Reply:
Okay I see the benefit of setting it a little higher to allow things to queue up. I understood that with it's current setting, it could be blocking if the max cost is repeatedly used

I'm thinking of just using a hardcoded default of 3? And just monitoring metrics to see how things look

Reply:
Metrics are awesome.